tests/unit/amo/sagas/test_collections.js

```diff
@@ -523,13 +524,19 @@ describe(__filename, () => {
     });
 
     describe('create logic', () => {
+      const params = {
+        description: { 'en-US': 'Collection description' },
+        name: { 'en-US': 'Collection name' },
+        slug,
+        user,
+      };
+      const collectionDetail = createFakeCollectionDetail();
```

Comment:
You can pass `params` to `createFakeCollectionDetail()` directly if I remember correctly.